### PR TITLE
Added Sentry to logError

### PR DESCRIPTION
**Pull request by mariocimet**

### CODE REVIEW

File: lib/helpers.js [hunk 1]
```diff
@@ -4,6 +4,7 @@
 // consideration. We want to avoid scope creep in this library that would require
```
Comment (trisapeace) [inline lib/helpers.js:4]: Could you please update the CHANGELOG?

Comment (trisapeace) [inline lib/helpers.js:4]: How does this know which Sentry instance? Do we need specific environment variables? Please add any necessary instructions to the README.

**Review by trisapeace — APPROVED** [review]
With changes that we said out loud.

File: lib/helpers.js [hunk 2]
```diff
@@ -42,6 +56,16 @@ function logError(logString) {
...
+    Sentry.captureException(logString)
+  }
+}
+
+function logSentry(logString) {
+  if (isTestEnvironment()) {
+    // Output in colour in test
+    console.log(chalk.dim.cyan(`\t${logString}`)) // eslint-disable-line no-console
+  } else {
+    Sentry.captureException(logString)
```
Comment (schwarrrtz) [inline lib/helpers.js:68]: @mariocimet @trisapeace does Sentry have a way to distinguish between log messages and error messages? is `captureException` the only way to submit logs to Sentry?

Comment (trisapeace) [reply lib/helpers.js:68]: Looks like capturing messages is also a thing: https://docs.sentry.io/platforms/node/usage/#capturing-messages

**Pull request merged**